src/rp2_common/pico_btstack/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ if (DEFINED ENV{PICO_BTSTACK_PATH} AND (NOT PICO_BTSTACK_PATH))
     message("Using PICO_BTSTACK_PATH from environment ('${PICO_BTSTACK_PATH}')")
 endif ()
 
+if (NOT PICO_BTSTACK_VERSION)
```

Comment:
How is the user supposed to use this; I assume they are supposed to set it if they are using a different version of btstack? It needs have PICO_CMAKE_CONFIG in that case, however it feels a bit odd that we expect them to set it, if it doesn't actual cause anything to happen e.g. choosing a version (if we just care about what version we have, can we determine it?)

Reply:
Yes, they'd define this if they want to use this cmake file to build an older version of btstack than the version in the sdk.

> it feels a bit odd that we expect them to set it, if it doesn't actual cause anything to happen
Sorry, I don't follow this comment. It would cause the build to not fail.

Reply:
> can we determine it?

Yes, maybe that's better. We can infer it's an older version by seeing if the file is there. I'll try that out.

Reply:
Ideally, all information about BTstack could/should be contained in the BTstack repo and Pico SDK would only refer to it. I'm not too familiar with your build system, but you may make some suggestions on how this could look like. E.g. we could stash Pico related files under port/pico-sdk, or just have CMakeLists.txt files that contains source files and include paths.

Reply:
I've pushed a version that just checks if the file exists. 

> Ideally, all information about BTstack could/should be contained in the BTstack repo

The ideal is to just include a file in the btstack repo. But arguably, it's too late now?